src/components/layout.js

```diff
@@ -35,6 +36,7 @@ const Layout = ({ children }) => {
         />
         {children}
       </div>
+      <Spacer heightDesktop="170px" heightMobile="80px" />
```

Comment:
I don't understand what this is for? There's nothing like this in the design, and I don't think we should be adding space to the Layout component like this because you won't be able to control it/undo it on a per page basis.

Reply:
ok